src/zino/zino.py

```diff
@@ -36,7 +36,7 @@ def init_event_loop(args: argparse.Namespace):
     scheduler.add_job(func=state.state.dump_state_to_log, trigger="interval", seconds=30)
 
     loop = asyncio.get_event_loop()
-    echo = loop.create_server(lambda: ZinoTestProtocol(), "127.0.0.1", 8001)
+    echo = loop.create_server(lambda: ZinoTestProtocol(state=state.state), "127.0.0.1", 8001)
```

Comment:
There's `adm_state`, `op_state` and now `zino.state`. Maybe change the latter name, is it for  server state?

Also, what is `echo` used for, is it unfinished?

Reply:
`adm_state` and `op_state` are attributes of an event/case.

`zino.state` is a module that holds the global/singleton root state object for the server, correct.  The global state object in that module is also named `state`.  As @stveit pointed out, this does turn into a "state soup" (e.g. `state = state.state`) and can get hard to read.

I've no problem with renames, but we need to agree on a naming system then.  I could live with the module `zino.server_state`, but is the object `zino.server_state.state` any better?

> Also, what is `echo` used for, is it unfinished?

Haha, indeed.  The initial code set up a simple echo server just for exploring how to code a server protocol using asyncio.  I guess the name was left in there by mistake once the code changed.

Reply:
Also, I'd say, this is hardly the PR to rename those things, but I will fix the echo-confusion :)